aidants_connect_habilitation/tests/test_views.py

```diff
@@ -749,6 +749,9 @@ def setUpTestData(cls):
             manager=ManagerFactory()
         )
         AidantRequestFactory(organisation=cls.organisation)
+        cls.organisation_no_manager: OrganisationRequest = (
+            DraftOrganisationRequestFactory()
```

Comment:
```suggestion
            DraftOrganisationRequestFactory(manager=None)
```

Juste pour être sûr de quoi on parle et au cas où la config de `DraftOrganisationRequestFactory` change pour une raison ou une autre.